article/_posts/2019-09-26-VQ-VAE.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: review
+title: "Neural Discrete Representation Learning"
+tags: deep-learning VAE
+author: "Jon Haitz Legarreta Gorroño"
+cite:
+    authors: "Aaron van den Oord, Oriol Vinyals, Koray Kavukcuoglu"
+    title:   "Neural Discrete Representation Learning"
+    venue:   "NIPS 2017"
+pdf: "https://arxiv.org/pdf/1711.00937.pdf"
+---
+
+
+# Highlights
+
+- Authors propose a Vector Quantised-Variational AutoEncoder (VQ-VAE), a variant
```

Comment:
quantized

Reply:
I used "Quantised" because authors used that spelling, at least for the method name. I guess they chose to use British English.